### ISSUE TITLE
Instructions for badge switching on repo ownership change

### ISSUE THREAD

**Issue by noamross** [ropensci/onboarding-meta#14]
- The editor's guide needs a set of instructions for editors/assisting editors/authors on how to switch over CI services and badges correctly.
- Need an "accepted" template (see RefManageR issue for example) to post into issue on acceptance with all author to-dos (inc. optional JOSS ones)
- Need to make sure editors all have the right permissions to make these switches on services
- Ideally this could be set up so authors can make the switch once they have admin permission for the repo.

**Comment by maelle** [comment]
Related to this, we need a few sentences about what opencpu CI is cc @jeroen. I've started receiving opencpu emails for my onboarded packages without really knowing about it.

**Comment by sckott** [comment]
> I've started receiving opencpu emails for my onboarded packages without really knowing about it

right, i think jeroen switched it on for all repos under `/ropensci`

AFAICT when a repo gets transferred to `ropensci` org, Travis does the switch automagically. But I don't think Appveyor does it automatically.

**Comment by karthik** [comment]
> Need to make sure editors all have the right permissions to make these switches on services

This would be great to add as a task for the buffy bot. 
`@rowbot authorize @user for appveyor`

**Comment by karthik** [comment]
@maelle We need to wrangle those opencpu build emails to one address. For now it went to info@ropensci but we'll have to find another place for it.

**Comment by jeroen** [comment]
I'll just make an option in opencpu to only notify the committer, not the repo owner.

**Comment by maelle** [comment]
@jeroen I think we also need to tell maintainers that it exists because if they commit, then they'll get an email, right? 

We have so much stuff to tell them that when their package gets onboarded the package author will receive a small book. :grin:
